spec/features/authenticate_admin_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'authenticated admin user', type: :feature do
+
+  before(:each) do
+    User.create(full_name: "vanessa", display_name: "admin", email: "admin@email.com", role: 1, password: "dinnerdash")
+    visit login_path
+  end
+
+  it 'can login as an admin' do
+    page.fill_in('Email', with: 'admin@email.com')
+    page.fill_in('Password', with: 'dinnerdash')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.